base/docs/basedocs.jl

```diff
@@ -693,4 +693,23 @@ run(pipeline(`cat test.txt`, DevNull))
 """
 DevNull
 
+"""
+    Function
+
+Abstract type for which all functions are instances. This includes operators, functions
```

Comment:
"Abstract type for which every function, operator, and lambda is a concrete subtype."? Or is that too dense?